src/transport/SessionManager.cpp

```diff
@@ -785,6 +806,26 @@ void SessionManager::ExpiryTimerCallback(System::Layer * layer, void * param)
     mgr->ScheduleExpiryTimer(); // re-schedule the oneshot timer
 }
 
+
+void SessionManager::ShiftToSession(const SessionHandle & handle)
+{
+    VerifyOrDie(handle->IsSecureSession());
+    SecureSession * session = handle->AsSecureSession();
+    VerifyOrDie(session->GetSecureSessionType() == SecureSession::Type::kCASE);
+    ScopedNodeId node = session->GetPeer();
```

Comment:
Suggest naming newNode

Reply:
There is only one peer, so I'll rename it to `peer`